core/backend/src/test/TestUtils.ts

```diff
@@ -104,3 +104,24 @@ after(async () => {
   await TestUtils.shutdownBackend();
 });
 
+/**
+ * Generate SHA1 Hash of Schema XML
+ * @param schemaXmlPath: Path where schema xml file is located.
+ * @param referencePaths: Schema reference paths.
+ * @param isExactMatch: Schema references are located by exact scheme version comparisons
+ */
+export function getSchemaSha1Hash(schemaXmlPath: string, referencePaths: string[], isExactMatch: boolean = false): string {
+  const ecSchemaOpsNative = NativeLibrary.load();
```

Comment:
Why do we have to do NativeLibrary.load here and delete  the library in the finally block?

Reply:
Looks like compensating for failing to call IModelHost.startup.
The tests shouldn't be using NativeLibrary directly anyway - there should be an API in core-backend that exposes this functionality to other packages.

Reply:
ok I have updated the tests now